Contributors.md

```diff
@@ -129,12 +144,17 @@ The following people have contributed to the SCAP Security Guide project
 * Leland Steinke <leland.j.steinke.ctr@mail.mil>
 * Justin Stephenson <jstephen@redhat.com>
 * Brian Stinson <brian@bstinson.com>
+* Jake Stookey <jakestookey@gmail.com>
+* Jonathan Sturges <jsturges@jsturges.remote.csb>
 * Philippe Thierry <phil@reseau-libre.net>
+* Derek Thurston <thegrit@gmail.com>
+* tianzhenjia <jiatianzhen@cmss.chinamobile.com>
 * Paul Tittle <ptittle@cmf.nrl.navy.mil>
 * tomas.hudik <tomas.hudik@embedit.cz>
 * Jeb Trayer <jeb.d.trayer@uscg.mil>
 * Matěj Týč <matyc@redhat.com>
 * VadimDor <29509093+VadimDor@users.noreply.github.com>
+* vojtapolasek <krecoun@gmail.com>
```

Comment:
This is duplicate of @vojtapolasek :smile:

Reply:
That is him.
Where is the duplicate in the list?

Reply:
@jan-cerny I see it, now! Fixed.